### PR TITLE
Added text after a colon in install.md (there was none)

### PR DESCRIPTION
**Pull request by aMySour**
I'm guessing there was meant to be text there, so I added some to explain how to do it

### CODE REVIEW

File: content/en/admin/install.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ menu:
 * A **domain name** \(or a subdomain\) for the Mastodon server, e.g. `example.com`
 * An e-mail delivery service or other **SMTP server**
 
-You will be running the commands as root. If you aren’t already root, switch to root:
+You will be running the commands as root. If you aren’t already root, switch to root: `sudo su -`
```
Comment (samwilson) [inline content/en/admin/install.md:16]: Is the hyphen always required? I think I've always done this with just `sudo su`.

Comment (Alex-Sour) [reply content/en/admin/install.md:16]: not sure, but it works for me

**Review by samwilson — APPROVED** [review]
@aMySour I'm just a random passer-by, but this looks fine to me! I only asked about the hyphen because it seems simpler without, but it's hardly a big deal. :-)

**Review by helix2301 — APPROVED** [review]
Adding the command to the end of the documentation feels logical and adds context so the reader does not need to look up the command.

**Review by Gargron — APPROVED** [review]

**Pull request merged**